### ISSUE TITLE
include metadata info from chapters' YAML blocks

### ISSUE THREAD

**Issue by trenkert** [otherjoel/simplebookmachine#1]
Is there a way to have the template read a YAML block in each different post0X.md file and pare it in the final document as titles, headers, footers  on a per-chapter basis?

**Comment by otherjoel** [comment]
Currently the script _does_ read the YAML block in each`.md` file and use that information in the titles (see [this section in the README](https://github.com/otherjoel/simplebookmachine#creating-and-formatting-your-chapters)). 

If you want to make use of YAML values other than title, author or date, you can add code for them in `latex-templates/chapter-tex.md` (i.e., pandoc templating code which outputs LaTeX code) similar to what already exists in that file.

**Comment by trenkert** [comment]
Yeah, I've tried that but it didn't work. The yaml block was ignored...
